AuroraEditor/Core/TabBar/UI/TabBar/TabBar.swift

```diff
@@ -234,40 +234,22 @@ struct TabBar: View {
                 )
                 // To fill up the parent space of tab bar.
                 .frame(maxWidth: .infinity)
-                .background {
-                    if prefs.preferences.general.tabBarStyle == .native {
-                        TabBarNativeInactiveBackground()
-                    }
-                }
             }
             // Tab bar tools (e.g. split view).
             trailingAccessories
         }
         .frame(height: TabBar.height)
-        .overlay(alignment: .top) {
-            // When tab bar style is `xcode`, we put the top divider as an overlay.
-            if prefs.preferences.general.tabBarStyle == .xcode {
```

Comment:
Wouldn't this need to stay tho?

Reply:
nope, the code within the if contains a if for not Xcode 😅

Reply:
Okay cool